cpp/CMakeLists.txt

```diff
@@ -504,6 +504,7 @@ install(TARGETS ${CUML_CPP_TARGET}
         DESTINATION lib)
 
 install(DIRECTORY ${CUML_INCLUDE_DIR}/cuml DESTINATION include)
+install(DIRECTORY ${RAFT_DIR}/cpp/include/ DESTINATION include/cuml)
```

Comment:
@wphicks would it be possible to add a note in the c++ readme of how to link/include libcuml.so in consuming projects?

Reply:
Yes, of course! I'll go ahead and add that.